WcaOnRails/app/controllers/registrations_controller.rb

```diff
@@ -221,7 +221,8 @@ def do_add
     registration_row[:wca_id]&.upcase!
     registration_row[:email]&.downcase!
     person_details = {
-      name: registration_row[:name],
+      first_name, last_name = registration_row[:name].split(' ', 2),
```

Comment:
Is this supposed to be in this PR?

Reply:
No, it's getting randomly added for whatever reason

Reply:
Then just rebase on master and make sure that it's not included in this PR :)